### PR TITLE
ikas-international-end-client-is-tier1-investment-bank-python-develop…

### PR DESCRIPTION
**Pull request by fadi78**
…er-with-flask-javascript-and-sql.html

.

### CODE REVIEW

**Comment by stestagg** [comment]
Hi Fadi

Unfortunately we only accept job listings where the hiring company (the place the candidate will actially work) is listed

Thanks

Steve

File: ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          # Choose one of the following options
+contract: contract
+contact:
+    name: Fadi Harb
+    email: fharb@ikasinternational.com
+    phone: 02072209344
+    # <additional contact fields>: <additional value>
+created: !!timestamp 2018-04-25
+tags:
+  - Python
```
Comment (salimfadhley) [inline ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html]: This seems like a great role, but can you provide more details about the skills required? For example, almost any web-based project will require some infrastructure knowledge unless it's purely an internal project. All JavaScript projects require some knowledge of JavaScript build-tooling such as NPM, Gulp, Grunt.

File: ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Python Developer with Flask - Javascript and SQL
+company: Ikas International (End client is Tier1 Investment Bank)
```
Comment (salimfadhley) [inline ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html]: This site requires that you disclose the name of the client. It's actually one of the few rules we have here.

File: ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - Flask
+  - Javascript
+  # include relevant tags
+---
+
+Python Developer - Flask, SQL, JavaScript
+
+Our client, a tier1 Investment Bank is currently seeking to hire a contract Python Developer to join a team that is currently building undertaking a new greenfield project to expand the capabilities of a current framework used for calculating risk.
+
+The role will sit with a Small team and will involve design, development and testing of developed applications. In order to be considered for this role, candidates must have the following skills:
```
Comment (salimfadhley) [inline ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html]: Can you say more about the role, for example, will the developer be expected to take part in a SCRUM type workflow, or something else? Are you looking for a junior developer or somebody with a great deal of experience? Clarity will help you find the right candidate.

**Comment by fadi78** [comment]
Hi Steve,

Thanks for the clarification, I really appreciate it. In our T&Cs with the client we’re not allowed to put that info when posting adverts.

However, we do share the name of the end client along with the job spec once someone applies, so the candidate will know where they’ll be working, but I can’t make that info public that’s all the only difference.

Would there any exceptions in our case or is that something completely fixed and can’t be changed?

Thanks

Fadi
Fadi Harb
Consultant
FHarb@ikasinternational.com   |  +44 7917 785 245

**Comment by stestagg** [comment]
Hey Fadi

Unfortunately this is a fixed rule.

Regards

Steve

**Comment by salimfadhley** [comment]
@fadi78 - I'm going to reject this ad for now. Feel free to contact me directly if you need help. As @stestagg mentioned, we have one rule on this site which is that users are obliged to reveal the employer. I'm sorry if your client does not permit this.

**Pull request closed without merging**